src/test/java/RestAssuredTests/TestGet.java

```diff
@@ -0,0 +1,45 @@
+package RestAssuredTests;
+
+import APICalls.GetCall;
+import io.restassured.RestAssured;
+import io.restassured.response.Response;
+import io.restassured.specification.RequestSpecification;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import utility.PropertyManager;
+
+import static org.hamcrest.Matchers.equalTo;
+
+
+import java.io.IOException;
+
+public class TestGet extends GetCall {
+    public TestGet() throws IOException {
+    }
+
+
+    static RequestSpecification httpRequest = RestAssured.given().when().header("Content-Type", "application/json");
+    static Response response = httpRequest.get(GET_URL);
+
+    @Test
+    public static void testId() throws IOException {
+        response.then().assertThat().body("id", equalTo(PropertyManager.getId()));
```

Comment:
ensure that each test is independent, your current setup might lead to tests affecting each other's outcomes
tests should ideally set up their own request and response objects